media/js/firefox/new/all-test-variations-pt2.js

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, dataLayer) {
+    'use strict';
+
+    // check page version for tests
+    var version = Number($('#masthead').data('version'));
+
+    // adds sort value & (optional) new text to direct download li
+    var prepLi = function(li) {
+        var $li = $(li);
+        var $liA = $li.find('a');
+        var newOsName;
+
+        switch ($liA.text()) {
+        case 'Windows':
+            newOsName = 'Windows 32-bit';
+            $li.attr('data-sort', 1);
+            break;
+        case 'Linux':
+            newOsName = 'Linux 32-bit';
+            $li.attr('data-sort', 2);
+            break;
+        case 'Windows 64-bit':
+            $li.attr('data-sort', 3);
+            break;
+        case 'Linux 64-bit':
+            $li.attr('data-sort', 4);
+            break;
+        case 'OS X':
+            $li.attr('data-sort', 5);
+            break;
+        default:
+            $li.attr('data-sort', 99);
+            break;
+        }
+
+        if (newOsName) {
+            $liA.text(newOsName);
+        }
+
+        return li;
+    };
+
+    var setupTest = function(version) {
+        var $dlButton = $('#download-button-desktop-release');
+        var $html = $('html');
+
+        // make sure desktop download button exists, user is not on Android or iOS, and user is on a recognized platform
+        if (!$html.hasClass('ios') && !$html.hasClass('android') && $dlButton.length && $dlButton.find('.unrecognized-download:visible').length === 0) {
```

Comment:
I think the reason we went desktop only here was to make sure the links are actionable. Clicking on the Google Play button while on iOS would probably be more frustrating than helpful.

I agree we may want to promote/display information about Android on iOS (and vice versa), but I think that's a different task than we're performing here.

(FWIW, my preference/hope is for a single link to a freshened up `/firefox/all/` instead of the handful of links we're testing here.)